frontend/micro-ui/web/micro-ui-internals/packages/modules/Contracts/src/pages/employee/CreateWorkOrder/index.js

```diff
@@ -102,7 +102,7 @@ const CreateWorkOrder = () => {
     const createDocumentObject = (documents) => {
         let docs =  documents?.filter(document=>document?.fileStoreId)?.map((document) => {
             return {
-                title: document?.fileType,
+                title: document?.fileType==="Others"?document?.fileName:document?.fileType,
```

Comment:
@nipunarora-eGov  Instead of checking for file type others
check this flag 
```showTextInput```
https://github.com/egovernments/works-mdms-data/blob/fc651fd186c80d985939822e587d234c19c8b922/data/pg/works/DocumentConfig.json#LL238C12-L238C25

Reply:
@jagankumar-egov This is for View. In Create Contract we are showing docs updloaded in estimates. Here I am setting proper title so that we can view the other file name in create contract screen